### PR TITLE
README changes

### PR DESCRIPTION
**Pull request by Zales0123**
Added solution how to register services in Symfony2 to use Sylius API Client.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,49 @@ $client = \Sylius\Api\Client::createFromUrl('http://demo.sylius.org/api/', $apiR
 
 ```
 
+If you use Symfony2 framework, you can also register some services, initialize Sylius API Client and inject it to other services.
+
+```yml
+
+parameters:
+    app.http_client.sylius.oauth_token.url: 'http://demo.sylius.org/oauth/v2/token'
+    app.sylius_client.id: secret_demo_client
```
Comment (michalmarcinkowski) [inline README.md]: `app.sylius_client.id: 1_demo_client`

File: README.md [hunk 2]
```diff
@@ -41,6 +41,49 @@ $client = \Sylius\Api\Client::createFromUrl('http://demo.sylius.org/api/', $apiR
...
+    app.sylius_client.secret: RuTbGMaH
```
Comment (michalmarcinkowski) [inline README.md]: `app.sylius_client.secret: secret_demo_client`

**Comment by michalmarcinkowski** [comment]
Thank you! :+1: 

Please just add a one line comment before each service to explain what is it.

**Comment by pjedrzejewski** [comment]
Thank you Mateusz! :+1:

**Pull request merged**